src/css.js

```diff
@@ -134,23 +135,24 @@ function getWidthOrHeight( elem, dimension, extra ) {
 	}
 
 
-	if ( ( isIE &&
-		(
-
-			// Support: IE 9 - 11+
-			// Use offsetWidth/offsetHeight for when box sizing is unreliable.
-			// In those cases, the computed value can be trusted to be border-box.
-			isBorderBox ||
-
-			// Support: IE 10 - 11+
-			// IE misreports `getComputedStyle` of table rows with width/height
-			// set in CSS while `offset*` properties report correct values.
-			nodeName( elem, "tr" )
-		) ||
+	if ( (
 
 		// Fall back to offsetWidth/offsetHeight when value is "auto"
 		// This happens for inline elements with no explicit setting (gh-3571)
-		val === "auto" ) &&
+		val === "auto" ||
+
+		// Support: IE 9 - 11+
+		// Use offsetWidth/offsetHeight for when box sizing is unreliable.
+		// In those cases, the computed value can be trusted to be border-box.
+		( isIE && isBorderBox ) ||
+
+		// Support: IE 10 - 11+
+		// IE misreports `getComputedStyle` of table rows with width/height
+		// set in CSS while `offset*` properties report correct values.
+		// Support: Firefox 70+
+		// Firefox includes border widths
+		// in computed dimensions for table rows. (gh-4529)
```

Comment:
Back when we had lots of support comment, when code was guarded by one, we usually didn't comment what gets worked around as the support comment above the support test was explaining that anyway. Maybe we can remove this comment in that case here.

Reply:
Although, I see we have a similar comment on `3.x-stable` here so I'm fine either way.